tests/plugins/borders.test.ts

```diff
@@ -533,6 +533,39 @@ describe("Grid manipulation", () => {
     expect(getBorder(model, "C96")).toEqual({ bottom: b });
     expect(getBorder(model, "D96")).toEqual({ right: b, bottom: b });
   });
+
+  test("Remove multiple rows => remove internal borders", () => {
```

Comment:
```suggestion
  test("Removing multiple rows removes the internal borders", () => {
```
nitpick. Try to do full sentences in test names where you can, it's more readable IMO